packages/nimble-components/src/table/template.ts

```diff
@@ -33,6 +33,7 @@ import {
 export const template = html<Table>`
     <template
         role="treegrid"
+        tabindex="0"
```

Comment:
As everything in the table will now start as tabindex=-1, this tabindex=0 ensures the table can be keyboard focused initially. The table remains as tabindex=0 for almost all (see `KeyboardNavigationManager.blurAfterLastTab()` for an exception)